.clang-format

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
I suggest reviewing all of these settings and ensuring they match what you and the rest of the team actually want. I've seen a few that might not be desirable.